abdulfatai360/politico
# #163598257 Setup ESLint Library

#### What does this PR do?
Setup linting library to serve as a guide for consistent code style and quality

#### Description of Task to be completed?
Configure .eslintrc file
Use airbnb style guide

#### How should this be manually tested?
Clone the repo and ```cd``` into it
Create ```anyfile.js``` and enter ```var x = 4```
```var``` should be flagged

#### What are the relevant pivotal tracker stories?
#163598257
